plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineProjectWizard.java

```diff
@@ -40,7 +40,7 @@
 
 public abstract class AppEngineProjectWizard extends Wizard implements INewWizard {
 
-  private AppEngineWizardPage page = null;
+  protected AppEngineWizardPage page = null;
```

Comment:
While I was fixing this, I noticed that we explicitly disable the _Finish_ button on the Cloud SDK validation errors pages, so it can never get to `performFinish()`. I'll leave this field `private`.